script/prepare-release.js

```diff
@@ -166,12 +173,37 @@ async function runReleaseBuilds () {
   })
 }
 
+async function verifyNewVersion () {
+  let newVersion = await getNewVersion(true)
+  let response = await promptForVersion(newVersion)
+  if (response !== 'Y') {
```

Comment:
What about `if (reponse.match(/^y/i)` to allow for "Yes" and "yes" and "Y" and "y"?

Reply:
@zeke I suppose so.  Though you could argue that having to type an exact character should prevent accidentally saying yes.

Reply:
An accidental yes seems unlikely since they still have to type 'y' + Enter ...?

I like Zeke's idea of catching all the 'yes' variations

Reply:
@ckerr @zeke I updated to use Zeke's recommendation